docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -38,8 +38,21 @@ If you have installed OpenVINO Runtime via the installer, APT, or YUM, follow th
 
 3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
 
-You've completed all required configuration steps to perform inference on processor graphics. 
-Proceed to the <a href="openvino_docs_install_guides_installing_openvino_linux.html#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+You are done updating your device driver and are ready to use your GPU.
+
+## What’s Next?
```

Comment:
I mean, we only need one "What's Next", because their content is identical. It can be put at the same level as Linux and Windows.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/12222/commits/10927d3a6de7276a92710fc113cd2999c032875a